cooker/bin/commands/start/request-listener.js

```diff
@@ -144,7 +144,7 @@ function _respond(
  * @param {string} file
  */
 function _doesntExist(res, file) {
-  const prettyPath = path.relative(config.CWD, file);
+  const prettyPath = path.relative('/', file);
```

Comment:
Was this broken @liamegan ?

Reply:
It wasn't broken, but the site I chose to test this on (Loz) was using the old name for the config file which was causing all sorts of problems. This was my attempt to test it (I was trying to test whether the SP thought it was in the src folder or not), it prints out the full location of the file, rather than the file relative to the CWD, which doesn't yield relevant information if you're wondering about things above the CWD.

Either way, this shouldn't really have been included in the PR and I'll roll it back.